CMakeLists.txt

```diff
@@ -31,6 +31,12 @@ set(LLRI_DIR_SRC "${CMAKE_CURRENT_SOURCE_DIR}/legion/engine" CACHE INTERNAL "")
 set(LLRI_DIR_APPLICATIONS "${CMAKE_CURRENT_SOURCE_DIR}/applications" CACHE INTERNAL "")
 set(LLRI_DIR_APPLICATIONS_DEPS "${CMAKE_CURRENT_SOURCE_DIR}/applications/deps" CACHE INTERNAL "")
 
+if (NOT MSVC)
+	set(LLRI_COMPILER_FLAGS -Wall -Wextra -Wpedantic -Werror -Wno-c++98-compat -Wno-c++98-compat-pedantic)
```

Comment:
I would recommend turning off the c++03 11 and 14 compatibility ones too

Reply:
Hmm I didn't get any 03/11/14 warnings